### ISSUE TITLE
Missing typings (already added in attached file)

### ISSUE THREAD

**Issue by hussain5416** [mhammond/pywin32#1809]
Change the attached file extension to `.py` from `.csv`
(It's a `.py` file, Github won't allow to upload `.py` file, that's why I changed the extension)

[win32typing.csv](https://github.com/mhammond/pywin32/files/7703014/win32typing.csv)

**Comment by vernondcole** [comment]
It seem that a great deal of work was done here, but there is no explanation of what is intended to be done with the included file. 

I think that you may be intending to provide new or updated code for pywin32 -- in which case you should prepare a pull request. We do not have the procedure for that well documented. 
Start by reading [the GitHub explanation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request ) and then continue your work with a clone of [the main branch](https://github.com/mhammond/pywin32) which you get by hitting the green "Clone" button. 
If you need more help, feel free to ask.

**Comment by hussain5416** [comment]
@vernondcole I earlier thought about opening the pull request, but I couldn't find the file specified in this repo (`win32typing.py`). So I just changed the original file using the VS Code, after installing pywin32 using `pip install` command. And uploaded the changed file here, so those who know where to update that file do the work. If I knew where that typing file came from, I would have opened a PR for it.

I needed to update the typing file, because there were some missing typings in that file.
